lib/ravendb/documents/session/document_session.rb

```diff
@@ -563,5 +566,34 @@ def document_query(klass, index_klass: nil, index_name: nil, collection_name: ni
                         index_name: index_name,
                         collection: collection_name)
     end
+
+    def include(path)
+      MultiLoaderWithInclude.new(self).include(path)
+    end
+
+    def load_internal(klass:, ids:, includes:)
+      load_operation = LoadOperation.new(self)
+      load_operation.by_ids(ids)
+      load_operation.with_includes(includes)
+      command = load_operation.create_request
+      unless command.nil?
+        @request_executor.execute(command, session_info: @session_info)
+        load_operation.result = command.result
+      end
+      load_operation.get_documents(klass)
+    end
+
+    def number_of_requests
+      number_of_requests_in_session
+    end
+
+    # TODO
+    def load_new(klass, ids)
```

Comment:
Can we put `klass` arg as the last on the list? In Ruby similarly to Node.js (dynamicly typed langs in general) class is not that important as in Java. In Node.js class arguments are always optional - methods are using `Raven-Node-Type` or falling back to returning JS object literals if type from `Raven-Node-Type` is unknown (not registered within conventions). In Ruby as we talked we should return either Hash or Struct in such case.